specification/purview/data-plane/Azure.Analytics.Purview.Catalog/preview/2023-02-01-preview/purviewcatalog.json

```diff
@@ -0,0 +1,10037 @@
+      }
+    },
+    "SortType": {
+      "type": "string",
+      "title": "SortType",
+      "description": "to specify whether the result should be sorted? If yes, whether asc or desc.",
+      "enum": [
+        "NONE",
+        "ASC",
+        "DESC"
+      ],
+      "x-ms-enum": {
+        "name": "SortType",
+        "modelAsString": true
+      }
+    },
+    "Status": {
+      "type": "string",
+      "title": "Status",
+      "description": "Status of the entity - can be active or deleted. Deleted entities are not removed from Atlas store.",
+      "enum": [
+        "ACTIVE",
+        "DELETED"
+      ],
+      "x-ms-enum": {
+        "name": "Status",
+        "modelAsString": true
+      }
+    },
+    "Status_AtlasRelationship": {
+      "type": "string",
+      "title": "Status",
+      "description": "The enum of relationship status.",
+      "enum": [
+        "ACTIVE",
+        "DELETED"
+      ],
+      "x-ms-enum": {
+        "name": "Status_AtlasRelationship",
+        "modelAsString": true
+      }
+    },
+    "TimeBoundary": {
+      "type": "object",
+      "properties": {
+        "endTime": {
+          "type": "string",
+          "description": "The end of the time boundary."
+        },
+        "startTime": {
+          "type": "string",
+          "description": "The start of the time boundary."
+        },
+        "timeZone": {
+          "type": "string",
+          "description": "The timezone of the time boundary."
+        }
+      },
+      "title": "TimeBoundary",
+      "description": "Captures time-boundary details"
+    },
+    "TimeZone": {
+      "type": "object",
+      "properties": {
+        "dstSavings": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The value of the daylight saving time."
+        },
+        "id": {
+          "type": "string",
+          "description": "The ID of the timezone."
+        },
+        "availableIds": {
+          "type": "array",
+          "description": "An array of available IDs.",
+          "items": {
+            "type": "string"
+          }
+        },
+        "default": {
+          "$ref": "#/definitions/TimeZone"
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The display name of the timezone."
+        },
+        "rawOffset": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The raw offset of the timezone."
+        }
+      },
+      "title": "TimeZone",
+      "description": "The timezone information."
+    },
+    "TypeCategory": {
+      "type": "string",
+      "title": "TypeCategory",
+      "description": "The enum of type category.",
+      "enum": [
+        "PRIMITIVE",
+        "OBJECT_ID_TYPE",
+        "ENUM",
+        "STRUCT",
+        "CLASSIFICATION",
+        "ENTITY",
+        "ARRAY",
+        "MAP",
+        "RELATIONSHIP",
+        "TERM_TEMPLATE"
+      ],
+      "x-ms-enum": 
```

Comment:
Is the error response also an atlas thing?

Azure guideline https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#rest-error-response-body-structure

Reply:
Yes. This error is in Atlas format. 
![image](https://github.com/Azure/azure-rest-api-specs/assets/78060542/0af69a71-a064-464b-acea-a7409abab13a)